lib/agent.js

```diff
@@ -6,6 +6,10 @@
  * See the included LICENSE file for more details.
  */
 
+// Polyfill for `String.prototype.normalize`.
+// To be removed when Node 0.x is no longer supported.
+require('unorm')
```

Comment:
Do we really need this polyfill? It seems `normalize` is on string since v0.12. I'm 👍 in removing support for v0.10 in the next release.